### DISCUSSION TITLE
Attempting to install nvim-orgmode on LazyVim

### DISCUSSION THREAD

**Discussion by wolfv6** [nvim-orgmode/orgmode#762] [Q&A] [answered]
I am attempting to install nvim-orgmode on LazyVim.
I copied code into init.lua from https://github.com/nvim-orgmode/orgmode#installation > lazy.nvim (recommended)
But this message comes up when I open nvim:
 
Error detected while processing /home/wolfv/.config/nvim/init.lua:
E5113: Error while calling lua chunk: /home/wolfv/.config/nvim/init.lua:5: attempt to call a boolean value
stack traceback:
        /home/wolfv/.config/nvim/init.lua:5: in main chunk

The init.lau file is attached: [init.lua.txt](https://github.com/user-attachments/files/16009282/init.lua.txt)

I am new to neovim and don't know Lau.
The nvim-orgmode install is not working.
How can I trouble shoot this?  I appreciate your help.

NVIM v0.10.0
lazy.nvim chore(main): release 11.4.2

**Comment by kristijanhusak** [comment]
Follow instructions how to add plugin here https://www.lazyvim.org/configuration/plugins

So in this case, create a file in `lua/plugins/orgmode.lua` and add this:

```lua
return {
  'nvim-orgmode/orgmode',
  event = 'VeryLazy',
  ft = { 'org' },
  config = function()
    require('orgmode').setup({
      org_agenda_files = '~/orgfiles/**/*',
      org_default_notes_file = '~/orgfiles/refile.org',
    })
  end,
}
```

**Reply by wolfv6** [reply]
Thanks for the quick response.

I restored ~/.config/nvim/[init.lua.txt](https://github.com/user-attachments/files/16018269/init.lua.txt) and added ~/.config/nvim/lua/plugins/[nvim-orgmode.lua.txt](https://github.com/user-attachments/files/16018265/nvim-orgmode.lua.txt)
The nvim-orgmode installation does not seem to work.
None of these work:
- Open agenda prompt: \<Leader>oa
- Open capture prompt: \<Leader>oc
- In any orgmode buffer
  - Help: press g? for help
  - Folding headings: \<tab>

What else can I try?

**Reply by kristijanhusak** [reply]
It is a problem with your configuration. Send me the link to your full configuration.

**Reply by wolfv6** [reply]
Do you mean my entire ~/.config/nvim/ directory?

**Reply by kristijanhusak** [reply]
Yes, if you have it on the repository just share that.

**Reply by wolfv6** [reply]
Here is my ~/.config/[nvim](https://github.com/user-attachments/files/16018815/nvim.zip)/ directory

**Reply by kristijanhusak** [reply, marked as answer]
You forgot the `return`.

This needs to be in `nvim-orgmode.lua` file:

```lua
return {
  'nvim-orgmode/orgmode',
  event = 'VeryLazy',
  ft = { 'org' },
  config = function()
    require('orgmode').setup({
      org_agenda_files = '~/orgfiles/**/*',
      org_default_notes_file = '~/orgfiles/refile.org',
    })
  end,
}
```

**Reply by wolfv6** [reply]
That works!  I was missing the "return".

Thanks for your help.  Much appreciated.
